doc/classes/ParticleProcessMaterial.xml

```diff
@@ -187,6 +187,7 @@
 		<member name="emission_box_extents" type="Vector3" setter="set_emission_box_extents" getter="get_emission_box_extents">
 			The box's extents if [member emission_shape] is set to [constant EMISSION_SHAPE_BOX].
 			[b]Note:[/b] [member emission_box_extents] starts from the center point and applies the X, Y, and Z values in both directions. The size is twice the area of the extents.
+			[b]Note:[/b] Only in the editor, changing this property emits the [signal Resource.changed] signal.
```

Comment:
```suggestion
			[b]Note:[/b] When running in the editor, changing this property emits the [signal Resource.changed] signal.
```
Or similar, to me this wording can be read as if this property is editor exclusive

Reply:
Sounds good to me